src/main/scala/uk/gov/hmrc/ct/computations/CPQ17.scala

```diff
@@ -41,5 +41,5 @@ case class CPQ17(value: Option[Boolean]) extends CtBoxIdentifier(name = "Trading
 }
 
 object CPQ17 {
-  val lossReform2017 = LocalDate.parse("2017-04-01")
+  val lossReform2017 = LocalDate.parse("2017-03-31")
```

Comment:
Would this be better as a function. IE have a function that takes a date and returns true or false if loss reform applies.

```
def lossReform2017Applies(endDate: EndDate): Boolean = {
  endDate.value.after(LocalDate.parse("2017-03-31"))
}
```
This could be tested easily and better describes the expected behaviour.